examples/sample-app/src/blocks/text.js

```diff
@@ -0,0 +1,32 @@
+import * as Blockly from 'blockly/core';
+
+// Create a custom block called 'add_text' that adds
+// text to the output div on the sample app.
+// This is just an example and you should replace this with your
+// own custom blocks.
+const addText = {
+  'type': 'add_text',
+  'message0': 'Add text %1 with color %2',
+  'args0': [
+    {
+      'type': 'input_value',
+      'name': 'TEXT',
+      'check': 'String',
+    },
+    {
+      'type': 'input_value',
+      'name': 'COLOR',
+      'check': 'Colour',
+    },
+  ],
+  'previousStatement': null,
+  'nextStatement': null,
+  'colour': 160,
+  'tooltip': '',
+  'helpUrl': '',
+};
+
+// Create the block definitions for the JSON-only blocks.
+// This does not register their definitions with Blockly.
```

Comment:
Also add (here or at the top of the file) "This file has no side effects" to echo your comment in generators.